### ISSUE TITLE
About the reproduction result of SDPO on lcb_v6

### ISSUE THREAD

**Issue by SyncrnzdClk** [lasgroup/SDPO#14]
Hi,
Thanks for your great work! I really enjoy it!
I'm trying to reproduce your results on lcb_v6. I downloaded the data just as you described in the readme.

`python data/split_tests.py \
    --json_path datasets/lcb_v6.json \
    --output_dir datasets/lcb_v6`
`python data/load_dataset.py --dataset_name livecodebench/code_generation_lite-v6 --output_path datasets/lcb_v6.json`

And my model is Qwen3-8B, the same model in your paper. However, I found my result seems to be have a higher accuracy than the paper's.
Something seems to be wrong, but I don't know where😂.

<img width="1470" height="919" alt="Image" src="https://github.com/user-attachments/assets/06ecf8e5-1916-4ee0-8f09-146af4711dd5" />

**Comment by jonhue** [comment]
@SyncrnzdClk good question! In our code environment `acc` reports the percentage of passed test cases. You are looking for `score` 😉

**Comment by Gengsheng-Li** [comment]
@SyncrnzdClk Hi, sorry to bother you! I'm currently trying to reproduce the results on lcb_v6, but I'm running into a MemoryError from Ray during training. My machine has 256GB of RAM in total. I was wondering if you encountered any similar memory issues during your reproduction, and if so, how you resolved them? Thanks in advance! :-D

I used the default training setting.

**Comment by SyncrnzdClk** [comment]
@Gengsheng-Li Hi! No problem at all.

Actually I didn't encounter memory issues in my environment, but that's likely because I'm using my company's machine which has 2.2Ti of RAM, which is significantly larger than the 256GB you're working with.😂

**Comment by Gengsheng-Li** [comment]
@SyncrnzdClk Thanks for such a quick reply, I guess I need to lower the batch size or somthing else...😂

**Comment by jpq20** [comment]
> [@SyncrnzdClk](https://github.com/SyncrnzdClk) Thanks for such a quick reply, I guess I need to lower the batch size or somthing else...😂

Hi, Sorry to bother you. I am also encountering this memory error. Have you found a solution yet?

**Comment by huangst21** [comment]
Great job! Has the author attempted to conduct an official evaluation of the trained model on LiveCodeBench? @jonhue
